hack/logextractor.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+#
+# This script can be used to extract the failure cluster logs tarball from the PVC created for an install pod.
+#
+# Usage: ./logextractor.sh [LOGS_PVC_NAME] [DEST_DIR]
+#
+
+echo "apiVersion: extensions/v1beta1
+kind: Deployment
```

Comment:
Is there a reason why we cannot use a Pod directly instead of a Deployment?

Reply:
Well originally I did this because the instructions I was following added the volume via CLI after creation, which you can't do with a pod. But then this later became inline yaml so at this point no, I don't think there's a reason it can't be a pod, will switch.